### ISSUE TITLE
sourcemap warning

### ISSUE THREAD

**Issue by ritz078** [ngrx/store#180]
Angular cli : current master having webpack.

```
WARNING in ./~/@ngrx/core/compose.js
Cannot find source file '../../lib/compose.ts': Error: Can't resolve '../../lib/compose.ts' in '/Users/ritz078/projects/ng-housing/node_modules/@ngrx/core'
 @ ./src/main.ts 5:0-45
 @ multi main

WARNING in ./~/@ngrx/core/SyncSubject.js
Cannot find source file '../../lib/SyncSubject.ts': Error: Can't resolve '../../lib/SyncSubject.ts' in '/Users/ritz078/projects/ng-housing/node_modules/@ngrx/core'
 @ ./~/@ngrx/store/dispatcher.js 7:20-53
 @ ./~/@ngrx/store/index.js
 @ ./src/main.ts
 @ multi main

WARNING in ./~/@ngrx/core/operator/select.js
Cannot find source file '../../../lib/operator/select.ts': Error: Can't resolve '../../../lib/operator/select.ts' in '/Users/ritz078/projects/ng-housing/node_modules/@ngrx/core/operator'
 @ ./~/@ngrx/store/store.js 8:15-52
 @ ./~/@ngrx/store/index.js
 @ ./src/main.ts
 @ multi main
```

**Comment by mb-jenks** [comment]
+1

**Comment by MikeRyanDev** [comment]
This will be fixed in the next releases of ngrx/store and ngrx/core

**Comment by psyCodelist** [comment]
Still see the issue in 2.1.2.

**Comment by mmrath** [comment]
It would be nice to get this fixed. Lot of warnings

**Comment by MikeRyanDev** [comment]
@ngrx/core v1.0.2 published without sourcemaps to remove these errors.

**Comment by MikeRyanDev** [comment]
Reopening. Didn't realize sourcemaps were broken in the latest release of Store. 🔥

**Comment by robwormald** [comment]
ahh crap. meant to handle this in the last release. @MikeRyan52 are you inlining the sourcemaps or?

**Comment by MikeRyanDev** [comment]
@robwormald There's some magic incantation using `sourceMap`, `sourceRoot`, and `moduleRoot` to get the sourcemaps working correctly that I haven't had the time to fully investigate yet. As a quick fix I've temporarily skipped generating them in effects and core.

**Comment by Meligy** [comment]
@robwormald  @MikeRyan52 can you guys please see if this PR #200 is a good solution for this?

Thanks heaps for all the awesomeness :)

**Comment by ggranum** [comment]
Angular-cli 1.0.0-beta.11-webpack.3  silences these warnings: 
[https://github.com/angular/angular-cli/blob/master/CHANGELOG.md]()
- **build:** silence sourcemap warnings for vendors ([#1673](https://github.com/angular/angular-cli/issues/1673)) ([67098e0](https://github.com/angular/angular-cli/commit/67098e0))

**Comment by Meligy** [comment]
Silencing the warning is the the CLI is -of course- CLI specific. It doesn't affect people using [AngularClass/angular2-webpack-starter](https://github.com/AngularClass/angular2-webpack-starter) for example.

**Comment by mmrath** [comment]
@robwormald, @ MikeRyan52 is there a fix or workaround for this issue. I use many libraries but only ngrx has this issue

**Comment by MikeRyanDev** [comment]
@mmrath Sourcemaps should have been fixed in v2.2.0.

**Comment by mmrath** [comment]
@MikeRyan52 Thank you. I just upgraded and the warnings went away.
